docs/index.md

```diff
@@ -0,0 +1,8 @@
+![simple action pipeline logo](img/SAP-small.png)
+# simple-action-pipeline (sap)
+
+{% include-markdown "about.md" rewrite-relative-urls=true %}
```

Comment:
Why bother using include-markdown for the about page here? As far as I can see the about page is only used once, here. I think that the main utility of the include-markdown plugin is about re-using markdown files in multiple places.